src/Particle/LongRange/EwaldHandler3D.cpp

```diff
@@ -40,9 +40,9 @@ void EwaldHandler3D::initBreakup(ParticleSet& ref)
   app_log() << "  Sigma=" << Sigma << std::endl;
   Volume     = ref.Lattice.Volume;
   PreFactors = 0.0;
-  fillFk(ref.SK->KLists);
-  fillYkgstrain(ref.SK->KLists);
-  filldFk_dk(ref.SK->KLists);
+  fillFk(ref.SK->getKLists());
```

Comment:
I think practically everywhere KLists is has been touched.  Rename it.  KLists looks like a type that is a list of KList, `k_lists_`

KContainer clearly wants to be named KLists but couldn't because someone loved a member variable named KLists, great time to fix this as well.

Reply:
I have changed KLists to k_lists_ inside StructFact.
I'm following the current naming rule for functions. 
https://qmcpack.readthedocs.io/en/develop/developing.html#member-function-names